scripts/linters/js_ts_linter.py

```diff
@@ -220,6 +228,57 @@ def _compile_ts_file(self, filepath, dir_path):
             dir_path, os.path.basename(filepath).replace('.ts', '.js'))
         return compiled_js_filepath
 
+    def _check_any_type(self):
+        """Checks if the type of any variable is declared as any
+        in typescript files.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting any type check')
+            python_utils.PRINT('----------------------------------------')
+
+        files_to_check = self.all_filepaths
+        patterns_to_match = [r':\ *any', r'^\ *any']
```

Comment:
Could you explain these two a bit?

Reply:
Added comments. Does that make it clear?

Reply:
Yep, but these cases are both quite different one is used on every line the other just on some lines. So I would separate them into two variables.

Reply:
Done